lang/en.js

```diff
@@ -291,6 +291,7 @@ var theUILang =
  Toggle_details			: "Toggle details",
  Toggle_categories		: "Toggle categories",
  Delete_current_torrents	: "Delete current torrent(s)",
+ Quick_search   : "Quick search",
```

Comment:
Could you fix the spacing in the language files, so it matches the other items?

Reply:
Yes, I'd be glad to do so. Just want to make sure how do I do it properly. Do I fix the spacing and summit another commit, and the changes will be automatically added to this pull request?